### PR TITLE
Fixes #2615: Inconsistency in output of Close button

### LINKED ISSUE #2615: [A11y] Admin Add Pin Screen
## Current Output

https://user-images.githubusercontent.com/9396084/106889293-cf97cd00-670d-11eb-96ba-9834145cc169.mp4

https://user-images.githubusercontent.com/9396084/106889297-d1fa2700-670d-11eb-8ad5-a28f9801d2cb.mp4

## Issues Identified
- [x] There are two texts at top and learner requires two separate actions to output both. Confirmed: Current implementation is already correct.
- [x] There are two cases where pins do not match (a.) both are equal length but mismatch (b.) both are of unequal length. Confirmed: Current implementation is already correct.
- [ ] Inconsistency in output of **Close** button
In case of **Admin Add Pin** screen the close button outputs "Navigate Up, Button" and in **Admin Auth Pin** screen the output is "Close, Button". Use **Close, Button** at both places.
- [x] Inactive submit button outputs "SUBMIT, Button" instead it should output "SUBMIT, Button, disabled" (finished in #2626 )

These issues are same for mobile-landscape, mobile-portrait, tablet-landscape and tablet-portrait.

Note: To understand the above issue(s) completely, it is recommended that you setup Talkback and play with the app keeping it on and that will give you better context.

**Accessibility Guide**: https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide

### PR DESCRIPTION
**Pull request by nskirkey**
## Explanation
Fixes #2615: In case of Admin Add Pin screen the close button outputs "Navigate Up, Button" and in Admin Auth Pin screen the output is "Close, Button". Use Close, Button at both places. This PR fixes this so that in the Admin Add Pin screen the outputs "Close, Button."

## Checklist
- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [X] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [X] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [X] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [X] The PR is made from a branch that's **not** called "develop".
- [X] The PR is made from a branch that is up-to-date with "develop".
- [X] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by anandwana001** [comment]
@nskirkey Please complete all the installation steps: https://github.com/oppia/oppia-android/wiki#installation

File: app/src/main/java/org/oppia/android/app/profile/AdminPinActivityPresenter.kt [hunk 1]
```diff
@@ -38,6 +38,8 @@ class AdminPinActivityPresenter @Inject constructor(
     activity.title = activity.getString(R.string.add_profile_title)
     activity.supportActionBar?.setDisplayHomeAsUpEnabled(true)
     activity.supportActionBar?.setHomeAsUpIndicator(R.drawable.ic_close_white_24dp)
+    activity.supportActionBar?.setHomeActionContentDescription(R.string.admin_auth_close)
+
```
Comment (kartikeysaran) [inline app/src/main/java/org/oppia/android/app/profile/AdminPinActivityPresenter.kt:42]: Delete the blank lines here to pass the test

Comment (rt4914) [reply app/src/main/java/org/oppia/android/app/profile/AdminPinActivityPresenter.kt:42]: +1

**Review by rt4914 — CHANGES_REQUESTED** [review]
@nskirkey Along with above change add a test similar to this PR: https://github.com/oppia/oppia-android/pull/2753

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**